src/components/DestinationBox.jsx

```diff
@@ -0,0 +1,49 @@
+import React, {useState, useEffect} from "react";
+
+export default function DestinationBox({destination, onDelete}) {
+  //console.log(destination.name}
+  const [visited, setVisited] = useState(destination.visited || false)
+  
+  const markAsVisited = () => {
+    setVisited(visitedTrue => !visitedTrue); // toggle the state between true and false
+  }
+
+  if (!destination) {
+    return <div> No destination available. </div>
+  }
+
+  return (
+    <div>
+      <div style={{border: '1px solid black', padding: '10px', margin: '10px', borderRadius: '30px', maxWidth:'300px'}}>
```

Comment:
it is not considered good practice to use inline styles in React components, use a class instead.